src/components/Storefront/common/Item.js

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { VStack, Heading, Image } from '@chakra-ui/react';
+
+const Item = ({ name, image, price }) => (
+  <VStack>
+    <Image border="1px solid black" w={80} h={80} src={image} alt={name} />
+    <Heading as="h4" size="subtitle">
+      {name.toUpperCase()}
```

Comment:
not that it really matters, but I've always preferred transforming to uppercase using the css attribute `textTransform` rather than with JS

Reply:
Yeah, agreed - using `textTransform` makes it easier to replace `name` in this case since any field passed through the heading will be uppercase